plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcClient.java

```diff
@@ -215,6 +215,8 @@ default void setTableProperties(ConnectorSession session, JdbcTableHandle handle
 
     boolean supportsRetries();
 
+    boolean supportsMergeRetries();
```

Comment:
Do we need this method today? The only implementation `return false`, so let's maybe inline it for now and add it later when we know it is could be more useful.